web/src/admin/AdminInterface/AdminInterface.ts

```diff
@@ -1,12 +1,12 @@
 import { ROUTES } from "@goauthentik/admin/Routes";
-import { DEFAULT_CONFIG } from "@goauthentik/common/api/config";
```

Comment:
Everything below here is this: Because "common" is a grab-bag of things, they want '.js' suffixes on everything we pull out of it.  Sad, but true.  Only the last two entries at the end of this page have anything different, and they're the changes to `tsconfig` to understand that `@goauthentik/common` is no longer an alias to a path, it's the name of a package.